src/remotebuild/test/darwinCerts.ts

```diff
@@ -17,7 +17,7 @@
 /* tslint:disable:no-var-requires */
 // var require needed for should module to work correctly
 // Note not import: We don't want to refer to shouldModule, but we need the require to occur since it modifies the prototype of Object.
-var shouldModule = require("should");
+var shouldModule: any = require("should");
```

Comment:
I didn't know that. According to this issue: https://github.com/Microsoft/TypeScript/issues/596 it seems that the correct way to handle this might be:
import "should";  // just for side effect